creusot-contracts/src/std/num.rs

```diff
@@ -157,6 +157,35 @@ macro_rules! spec_op_common {
     };
 }
 
+#[logic]
+fn abs_diff(a: Int, b: Int) -> Int {
+    if a < b {
+        b - a
+    } else {
+        a - b
+    }
+}
+
+/// Adds specifications for the abs_diff operation on the given pair of signed
+/// and unsigned integer types
+macro_rules! spec_abs_diff {
```

Comment:
I don't think so because abs_diff requires knowing both the signed and the unsigned type (the signed variant of the function returns the unsigned type).

Reply:
Right, I was just looking at he contracts.